<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VmcDetailsActivity</class>
 <widget class="QWidget" name="VmcDetailsActivity">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>831</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="mp_layout_2">
     <item>
      <widget class="QPushButton" name="mp_btn_close">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
       <property name="icon">
        <iconset resource="../Resources/Resources.qrc">
         <normaloff>:/images/back</normaloff>:/images/back</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="OplPcTools::UI::ClickableLabel" name="mp_label_vmc_title">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>24</height>
        </size>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="mp_btn_rename">
       <property name="toolTip">
        <string>Rename...</string>
       </property>
       <property name="icon">
        <iconset resource="../Resources/Resources.qrc">
         <normaloff>:/images/rename</normaloff>:/images/rename</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="mp_frame">
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="mp_layout">
        <item>
         <widget class="QToolButton" name="mp_btn_fs_back">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Back</string>
          </property>
          <property name="text">
           <string notr="true"/>
          </property>
          <property name="arrowType">
           <enum>Qt::ArrowType::LeftArrow</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="mp_edit_fs_path">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="mp_widget_error_message" native="true">
        <property name="styleSheet">
         <string notr="true">#mp_widget_error_message {
    background-color: rgba(233, 67, 37, 50);
    border: 2px solid rgb(233, 67, 37);
}</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="mp_label_error_message">
           <property name="text">
            <string notr="true">Error message</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::TextInteractionFlag::LinksAccessibleByMouse|Qt::TextInteractionFlag::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QTreeView" name="mp_tree_fs">
        <property name="contextMenuPolicy">
         <enum>Qt::ContextMenuPolicy::CustomContextMenu</enum>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SelectionMode::ExtendedSelection</enum>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
        <property name="itemsExpandable">
         <bool>false</bool>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <property name="allColumnsShowFocus">
         <bool>true</bool>
        </property>
        <property name="expandsOnDoubleClick">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="mp_layout_3">
        <item>
         <widget class="QLabel" name="mp_label_charset">
          <property name="text">
           <string>Charset</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="mp_combobox_charset"/>
        </item>
        <item>
         <spacer name="mp_spacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QProgressBar" name="mp_progress_bar_free">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>240</width>
            <height>0</height>
           </size>
          </property>
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="mp_action_create_directory">
   <property name="icon">
    <iconset resource="../Resources/Resources.qrc">
     <normaloff>:/images/add</normaloff>:/images/add</iconset>
   </property>
   <property name="text">
    <string>Create Folder</string>
   </property>
   <property name="shortcut">
    <string>F7</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="mp_action_delete">
   <property name="icon">
    <iconset resource="../Resources/Resources.qrc">
     <normaloff>:/images/delete</normaloff>:/images/delete</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="mp_action_download">
   <property name="icon">
    <iconset resource="../Resources/Resources.qrc">
     <normaloff>:/images/download</normaloff>:/images/download</iconset>
   </property>
   <property name="text">
    <string>Download</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="mp_action_rename_entry">
   <property name="icon">
    <iconset resource="../Resources/Resources.qrc">
     <normaloff>:/images/rename</normaloff>:/images/rename</iconset>
   </property>
   <property name="text">
    <string>Rename</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="mp_action_upload_files">
   <property name="icon">
    <iconset resource="../Resources/Resources.qrc">
     <normaloff>:/images/file</normaloff>:/images/file</iconset>
   </property>
   <property name="text">
    <string>Files</string>
   </property>
   <property name="toolTip">
    <string>Upload Files</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="mp_action_upload_directory">
   <property name="icon">
    <iconset resource="../Resources/Resources.qrc">
     <normaloff>:/images/folder</normaloff>:/images/folder</iconset>
   </property>
   <property name="text">
    <string>Folder</string>
   </property>
   <property name="toolTip">
    <string>Upload Folder</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+U</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OplPcTools::UI::ClickableLabel</class>
   <extends>QLabel</extends>
   <header location="global">OplPcTools/UI/ClickableLabel.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mp_tree_fs</tabstop>
  <tabstop>mp_combobox_charset</tabstop>
  <tabstop>mp_btn_close</tabstop>
  <tabstop>mp_btn_rename</tabstop>
  <tabstop>mp_btn_fs_back</tabstop>
  <tabstop>mp_edit_fs_path</tabstop>
 </tabstops>
 <resources>
  <include location="../Resources/Resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>mp_btn_close</sender>
   <signal>clicked()</signal>
   <receiver>VmcDetailsActivity</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>126</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
